logger.go

```diff
@@ -18,8 +16,9 @@ var (
 )
 
 type Config struct {
-	Level zapcore.Level `required:"true" default:"warn"`
-	Debug bool          `required:"true" default:"false"`
+	Level       zapcore.Level `required:"true" default:"warn"`
+	Debug       bool          `required:"true" default:"false"`
+	ServiceName string        `required:"true" default:"default_service"`
```

Comment:
@lyschoening Also I didn't really know how to solve this. If we want to have a nice `service name` in the GCP error reporting we have to specify it (see line 52). So I decided to go with some default value and gradually provide `LOG_SERVICENAME` env variable for every service - that is backwards compatible + seemed like the least number of changes throughout the services

Reply:
Yes, that should work well for now.